plugins/module_utils/redfish_utils.py

```diff
@@ -2243,7 +2245,20 @@ def virtual_media_insert_via_patch(self, options, param_map, uri, data):
                   if k.endswith('@Redfish.AllowableValues'))
         # construct payload
         payload = self._insert_virt_media_payload(options, param_map, data, ai)
-        if 'Inserted' not in payload:
+        # check version of VirtualMedia schema
+        inserted_rw = True
+        match = re.search('#VirtualMedia\.v?(.*)\.VirtualMedia', data['@odata.type'])
```

Comment:
To make sure the backslashes don't need to be escaped, use a raw string:
```suggestion
        match = re.search(r'#VirtualMedia\.v?(.*)\.VirtualMedia', data['@odata.type'])
```

Reply:
Thanks, I have added the changelog fragment.